docs/MANUAL.md

```diff
@@ -34,6 +34,8 @@ _Copyright © 2018 Daniel Oaks <daniel@danieloaks.net>_
 - Commands
 - Integrating with other software
     - HOPM
+    - ZNC
```

Comment:
We should prolly remove ZNC from this list since it's more meant to be interacting with other software on the server-side. Tor makes sense though, it's similar to HOPM in that it interacts very directly on the server's side.

Reply:
My vision for this section was a general "here's how Oragono plays well or badly with these other pieces of software", e.g., it starts with a discussion of bots.

My first choice would just be retitling the section "Working with other software". I could split it into "Client support" and "Integrating with other software" (implicitly, server-side) sections, though.

Reply:
Ahh, right. Yeah. Sorry I forgot that this actually links to ZNC-specific information, ignore me.